global_prios/global_prios.csv

```diff
@@ -1,5 +1,5 @@
 Topic, Yearly QALY/DALY, Keywords,Reference
-Extreme poverty, 1e9, (poverty),https://80000hours.org/articles/problem-framework/
-Cure Cancer,1e8,(cancer,tumor,tumour),https://80000hours.org/articles/problem-framework/
-US land use,1e6,(US land use),https://80000hours.org/articles/problem-framework/
+Extreme poverty, 1e9, (poverty AND (international OR africa OR india OR (lower AND middle AND income)) ),https://80000hours.org/articles/problem-framework/
```

Comment:
spacings around commas are inconsistent - is this important?

Reply:
Since the table is being generated by humans, I think it's sensible for this to be dealt with in post-processing